docs/_guide/05-template-reference.md

```diff
@@ -532,4 +532,75 @@ const content = fetch('./content.txt').then(r => r.text());
 html`${until(content, html`<span>Loading...</span>`)}`
 ```
 
+## Rendering nothing
+Very often you may conditionally render something, but in the other case
+you do not want to render anything at all. So, if you are using ternary 
+operators in one case you might return your template and in the other you
+want to return no content lit-html leaves you with a couple options:
+* an empty string (`''`)
+* null
+* undefined
+* nothing sentinel
+### An empty string
+If you return an empty string somewhere in your template, an empty text node 
+will be rendered, but to the user it will seem like nothing was rendered. 
+
+````js
+${user.isAdmin
+      ? html`<button>DELETE</button>`
+      : ''
+  }
+````
+This does the trick and works great and efficiently in most cases. It does
+not work with slot fallback content.
+### null or undefined
+If you return `null` or `undefined` in your template, lit-html will use these as values. 
+This works most of the time, but may lead to unexpected behavior as the DOM itself does not
+handle the two equally:
+
+````js
+document.getElementById('container').innerHTML = null;`
+````
+will produce an empty div while
+````js
+document.getElementById('container').innerHTML = undefined;
+````
+produces a div with the text undefined in it. Also an attribute binding
```

Comment:
I think a better example (using lit) here might be that `<input value=${thing ? thing.prop : undefined}>` renders `'undefined'` as the value.

In this case `''` renders `value=""` and you can use `ifDefined` to not include the `value` attribute at all.

Reply:
I did add a new example with `undefined` as attribute value, but kept the old one too, as null does render no content when set through innerHTML and when setting the attribute to null it sets the text to null which is a difference in behavior the user might not expect. 

Also added a comment about `ifDefined` I have a feeling this might be more conditional rendering already.